packages/Radio/src/Radio.styles.js

```diff
@@ -0,0 +1,196 @@
+import { css } from "styled-components";
+import { toInt, fontSizeValue, lineHeightValue, z } from "@paprika/stylers/lib/helpers";
+import { boxSizingStyles, visuallyHidden } from "@paprika/stylers/lib/includes";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import tokens from "@paprika/tokens";
+
+const getLabelLeftPadding = (radioSize, hasLabel) => {
+  return hasLabel ? `${toInt(radioSize) + toInt(tokens.space)}px` : radioSize;
+};
+
+const smallRadioSize = tokens.radio.sizeSm;
+const mediumRadioSize = tokens.radio.sizeMd;
+const largeRadioSize = tokens.radio.sizeLg;
+
+const getHalfSizeCss = sizeCss => `${toInt(sizeCss) / 2}px`;
+const smallRadioHalfSize = getHalfSizeCss(smallRadioSize);
+const mediumRadioHalfSize = getHalfSizeCss(mediumRadioSize);
+const largeRadioHalfSize = getHalfSizeCss(largeRadioSize);
+
+const styles = {
```

Comment:
A lot of this file is copied from the `<Checkbox>`.  I don't mind the copy pasta as phase one, but do you have any ideas yet for how to DRY this up in a later PR perhaps?

Reply:
Not yet but I'll give it some thought in the next refactor where I remove any classes being applied on elements....